### PR TITLE
Code release from Staging to Depoloyment

### PR DESCRIPTION
**Pull request by PeteBoucher**

### CODE REVIEW

File: spec/views/static_pages/about.html.erb_spec.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+describe 'static_pages/about' do
+	it 'should include avatar images for our developers' do
+    pending
+		# render
+		# rendered.should have_selector('img#avatarThomas')
+	end
+end
```
Comment (yggie) [inline spec/views/static_pages/about.html.erb_spec.rb:9]: I might have merged this in a little prematurely, but as we have a client meeting tomorrow who we need to show something, I will overlook this pending test, but we must remember to implement it after tomorrow

**Comment by tochman** [comment]
Since we showed the client the about page with avatars and she rejected it on the basis of formatting we should focus on FIXING that rather than removing the content all together. Also, the way it looks now is not cool - with the footer pulled up all the way. 
skarmavbild 2014-04-01 kl 15 57 45

**Comment by yggie** [comment]
Sorry Thomas, I did not check the staging server manually (rookie mistake), however the code is there to produce something, might be an `rake assets:precompile` issue, did we set up Codealia to work like WebsiteOne (having Heroku precompile the assets?)

**Comment by tochman** [comment]
I think it is pulled up like this due to the fact that we do not have any content in that well. I don't think its a precompile issue - but I'm not 100% sure. 

Anyway, we should fill this page with some content - there is a [story for it](https://www.pivotaltracker.com/story/show/67496440), and we want to present a finished solution to the client 2morrow.

**Comment by yggie** [comment]
Lets work on it in the prep meeting later today (after the Atlantic scrum), it seems to be working locally though

**Comment by yggie** [comment]
Under the "files changed", there is something in the `about.html.erb` page, but it does not appear to be rendered. We still need to visit this anyway, because the "About Us" page doesn't look great locally

**Comment by yggie** [comment]
Ok sorry, another rookie mistake, the code was commented out...

**Pull request merged**